images/handlers.go

```diff
@@ -40,6 +40,10 @@ var (
 	// This applies only to a single descriptor in a handler
 	// chain and does not apply to descendant descriptors.
 	ErrStopHandler = fmt.Errorf("stop handler")
+
+	// ErrEmptyWalk is used when the WalkNotEmpty handlers return no
+	// children (e.g.: they were filtered out).
+	ErrEmptyWalk = fmt.Errorf("Empty Walk - image might be filtered out")
```

Comment:
* How about getting rid of `Empty Walk -`? Go's errors are usually not capitalized and `Walk` is foreign for developers who don't know containerd's internals.

Reply:
Done